qemu_v8.mk

```diff
@@ -40,6 +40,8 @@ EDK2_BIN		?= $(EDK2_PATH)/Build/ArmVirtQemuKernel-$(EDK2_ARCH)/$(EDK2_BUILD)_$(E
 QEMU_PATH		?= $(ROOT)/qemu
 QEMU_BUILD		?= $(QEMU_PATH)/build
 SOC_TERM_PATH		?= $(ROOT)/soc_term
+MODULE_OUTPUT		?= $(ROOT)/module_output
```

Comment:
I would prefer `$(ROOT)/out/kernel_modules` (note that `$(ROOT)/out` already exists).

Reply:
sure thing, will update in v3